unused/testdata/src/linkname/linkname.go

```diff
@@ -24,4 +24,4 @@ var (
 
 var ol int // used
 
-//go:linkname doesnotexist other5
+//go:linkname nonexistent other5
```

Comment:
Afaict the precise term here isn't important...

Reply:
> Afaict the precise term here isn't important...

That is true. But it also does not have to be a single, valid English word, either. See the concerns I've expressed on cache_test.go.